docs/development.rst

```diff
@@ -189,12 +189,19 @@ Prerequisites
 
 3. If you plan to contribute back to the project, see the `contributing guidelines <https://github.com/confluentinc/cp-docker-images/blob/master/CONTRIBUTING.md>`_.
 
-Adding Connectors to the Kafka Connect Image
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _adding_connectors_to_images :
 
-There are currently two ways to add new connectors to the Kafka Connect image.
+Adding Connectors to the Kafka Connect Images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Build a new Docker image that has connector installed. You can follow example 2 in the documentation below. You will need to make sure that the connector jars are on the classpath.
+Confluent provides two images for Kafka Connect:
+
+    - The Kafka Connect Base image is a minimal but easily-extended image that contains Kafka Connect and all of its dependencies. When started, it will run the Connect framework in distributed mode.
```

Comment:
think you can leave this part off: `is a minimal but easily-extended image` and `When started`, and simply say:

```
The Kafka Connect Base image contains Kafka Connect and all of its dependencies. 
It runs the Connect framework in distributed mode.
```